src/minicraft/entity/mob/Player.java

```diff
@@ -437,21 +437,11 @@ public void tick() {
 					level.dropItem(x, y, drop);
 			}
 			
-			if ((activeItem == null || !activeItem.used_pending) && (input.getKey("attack").clicked || input.getKey("pickup").clicked) && stamina != 0 && onFallDelay <= 0) { // this only allows attacks or pickups when such action is possible.

```

Comment:
why was this removed?

Reply:
oh I see, it was put elsewhere... I suppose that makes sense. But keep in mind that not addressing the used-pending status may result in some item bugs. Make sure you know what you're doing.